metplus/util/met_util.py

```diff
@@ -1694,7 +1694,7 @@ def getlist(list_str, expand_begin_end_incr=True):
         return []
 
     # FIRST remove surrounding comma, and spaces, form the string.
-    list_str = list_str.strip().strip(',').strip()
+    list_str = list_str.strip(';[] ').strip().strip(',').strip()
```

Comment:
Want to make sure this change is intentional. I assume it is, but this is the sort of the small change that could wreak havoc if not intended!

Reply:
Yes, this is the change that resolves #1212 by stripping off square braces and semi-colons from the list value.